.github/workflows/build.yml

```diff
@@ -1,26 +1,101 @@
 name: Build oee-simulators
 
 on:
-  workflow_call:
+  workflow_dispatch:
+    inputs:
+      environment:
+        type: environment
+        description: Name of the environment to use
+        default: "oee-dev"
+        options:
+          oee-dev: oee-dev Environment
+        required: true
+
+      release_body:
+        type: string
+        description: GitHub Release Message
+        default: "Release version ${GITHUB_RUN_NUMBER}"
+
+      run_release:
+        description: 'Run release workflow?'
+        required: false
+        default: false
+        type: bool
+
+      run_build:
+        description: 'Run build workflow?'
+        required: false
+        default: false
+        type: bool
+
+  push:
+    branches: [ '*' ]
 
 jobs:
+  call-test-workflow:
+    uses: ./.github/workflows/test.yml
+    with:
+      environment: 'oee-dev'
+    secrets:
+      C8Y_PASSWORD: ${{ secrets.C8Y_PASSWORD }}
+
+  recall-test-workflow-on-other-environment:
```

Comment:
why is the test workflow called twice?

Reply:
https://cumulocity.atlassian.net/browse/COE-910
I explained it in the new tasks that I created myself to test for a possibility